Cargo.toml

```diff
@@ -18,7 +18,7 @@ homepage = "https://github.com/Cardinal-Cryptography/drink"
 license = "Apache-2.0"
 readme = "README.md"
 repository = "https://github.com/Cardinal-Cryptography/drink"
-version = "0.8.6"
+version = "0.8.7"
```

Comment:
the only thing I'm concerned, is that whether this bump is backwards compatible... if there is any chance that this will break existing things, I would bump to `0.9.0`

cc: @deuszx

Reply:
rolling this back keeping that for next PR